roles/confluent.variables/vars/main.yml

```
@@ -504,6 +504,7 @@ kafka_connect_properties:
       producer.bootstrap.servers: "{{ groups['kafka_broker'] | default(['localhost']) | join(':' + kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']|string + ',') }}:{{kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']}}"
       consumer.bootstrap.servers: "{{ groups['kafka_broker'] | default(['localhost']) | join(':' + kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']|string + ',') }}:{{kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']}}"
       confluent.license.topic: _confluent-command
+      rest.extension.classes: "{% if kafka_connect_extension_classes_custom | length >0 %}{{ kafka_connect_extension_classes_custom }}{% endif %}"
```

I think in the case of non rbac, and an unchanged kafka_connect_extension_classes_custom, then the dictionary will then have `rest.extension.classes: ""` which we don't want. You can do this

```suggestion
  rest_classes:
      enabled: "{{ kafka_connect_extension_classes_custom||length > 0 }}"
      rest.extension.classes: "{{ kafka_connect_extension_classes_custom }}"
```

---

seems to require a properties object after enabled, following the other lists.